mne/decoding/time_gen.py

```diff
@@ -411,10 +396,12 @@ def score(self, epochs=None, y=None, scorer=None, test_times=None):
             need not be regular.
         """
 
-        from sklearn.metrics import (roc_auc_score, accuracy_score,
-                                     mean_squared_error)
-        from sklearn.base import is_classifier
-        from sklearn.preprocessing import LabelEncoder
+        from sklearn.metrics import accuracy_score
+
+        # Check scorer
+        if scorer is None:
+            scorer = accuracy_score
+        self.scorer = scorer
```

Comment:
If this attribute is not set at construction time it should end with an underscore.